### PR TITLE
Update Business Info (Also added keywords)

### PR DESCRIPTION
**Pull request by ctperry0301**

### CODE REVIEW

File: frontend/isoko/src/pages/BusinessDashboard/UpdateInfo.tsx [hunk 1]
```diff
@@ -1,8 +1,42 @@
...
+               link: 'https://isoko.com/',
+            },
+            {
+               title: 'Menu',
+               link: 'https://isoko.com/menu',
+            },
+         ]}
+         description="Bob rallies the team to cook up some business, so he sends middle child Gene to push something or other here this is long this is long this is long this is long this is long this is long this is long this is long this is long"
+         stars={3}
+         minorityTags={['Black-Owned']}
```
Comment (reillynski) [inline frontend/isoko/src/pages/BusinessDashboard/UpdateInfo.tsx:34]: v minor fix, but here and in the default value we replaced the hyphen with a space for minority groups

File: frontend/isoko/src/pages/ListBusiness.tsx [hunk 2]
```diff
@@ -178,24 +182,75 @@ const StyledCheck = styled(Form.Check)`
...
+   keywordTags: string[];
+   verified: boolean;
+   numReviews: number;
+   hours: Hours;
+   links: Link[];
+}
+
+const ListBusiness: React.FC<ListBusinessProps> = (props) => {
+   const [err, setErr] = useState('');
+   const [isOwner, setIsOwner] = useState(props.ownerDesc ? 'true' : '');
```
Comment (reillynski) [inline frontend/isoko/src/pages/ListBusiness.tsx:233]: in order to even be on this page they have to be the owner, right? so probably no need to use state here, maybe just set the box to be permanently checked and always have the owner desc visible

File: frontend/isoko/src/pages/ListBusiness.tsx [hunk 3]
```diff
@@ -215,10 +270,43 @@ const ListBusiness: React.FC = () => {
...
+      console.log('PH: ', props.hours);
+      if (props.hours !== undefined) {
+         Object.entries(props.hours).forEach((entry) => {
+            const [key, value] = entry;
+            x[key] = value.split('-')[0];
+            y[key] = value.split('-')[1];
+         });
+         setOpenHours(x);
+         setCloseHours(y);
+      }
```
Comment (reillynski) [inline frontend/isoko/src/pages/ListBusiness.tsx:302]: not sure if you already have this on the ListBusiness page, but might be worth doing a quick validation that they've entered an opening AND closing value

**Review by reillynski — CHANGES_REQUESTED** [review]
all in all looks great! just a few changes i mentioned in the comments. also wondering if we figured out where the user would be directed after they hit 'submit'? do they get navigated to the business preview tab where their changes are reflected? if not, might be worth having a small modal that says "changes saved" or something

File: frontend/isoko/src/pages/ListBusiness.tsx (hunk 2 above)
Comment (ctperry0301) [reply frontend/isoko/src/pages/ListBusiness.tsx:233]: The updateInfo form is the same component as the listBusiness form, so it won't always be accessed in the context of updateInfo. The isOwner would not false for a user who is listing a business on behalf of the owner.

File: frontend/isoko/src/pages/ListBusiness.tsx (hunk 3 above)
Comment (ctperry0301) [reply frontend/isoko/src/pages/ListBusiness.tsx:302]: props.hours (if included) will automatically determine both of these values. This function just sets the initial state of the component, so I dont think validation is needed here.

**Comment by ctperry0301** [comment]
> all in all looks great! just a few changes i mentioned in the comments. also wondering if we figured out where the user would be directed after they hit 'submit'? do they get navigated to the business preview tab where their changes are reflected? if not, might be worth having a small modal that says "changes saved" or something

I added an alert that basically says changes saved, but I don't navigate them anywhere. Might be worth discussing and potentially adding

**Review by reillynski — APPROVED** [review]
let's get this money

**Pull request merged**